src/component/AddItem.js

```diff
@@ -45,8 +48,12 @@ const AddItem = ({ firestore }) => {
         });
       } else {
         itemsDocRef.set({
+          id: normalizedName,
           name: name,
           numberOfDays: +numberOfDays,
+          dateOfPurchase: today,
+          numberOfPurchases: 1,
+          nextPurchaseDate: nextPurchase,
```

Comment:
This is all looking good, but I'd suggest here _not_ storing the next purchase date. A principle in React, generally, is that you don't store what you can calculate at render time. Storing the last purchase and the purchase interval (here called `numberOfDays`) is enough information to calculate `nextPurchaseDate` at the point where we're displaying the list of items. By not calculating it here, we keep this function a little simpler, we have to store a little less data, and we have to send a little less data over the network.

Reply:
Great! We can make that change. There's not really a place to put it right now though - I assume we'll eventually use it in ordering the list or some such thing. @segdeha should I just delete the parts where it's being added for now? 

Also, FYI - the AC actually does say to store it in the DB :) 
"When a purchase is recorded, a new estimated next purchase date should be calculated and recorded in the database"

Reply:
Ah haha haaaa, so it does 😅😂🤪

I will change the AC for the next cohort to be more clear. What really needs to be stored is the last purchase date and the estimated number of days until the next purchase. Thanks for reminding of what I wrote 😛

And, yes, with that info being properly stored in the DB, we'll use it in a future story that asks you to order the display of the list by how soon the item will need to be purchased again

Reply:
Haha 😄 Awesome - sounds good! I just removed that logic!